frontend/src/features/Semaphore/components/SearchSemaphoreButton/SearchSemaphores.tsx

```diff
@@ -100,7 +100,7 @@ export function SearchSemaphores() {
         onClose={onClose}
         onQuery={handleQuerySemaphore}
         options={optionsRef.current}
-        optionValueKey={'name' as any}
+        optionValueKey={'name' as string}
```

Comment:
```suggestion
        optionValueKey="name"
```

S'il y a des cas où ça ne passe pas c'est qu'il y a un soucis avec le typage (= on ne devrait pas avoir de cas où l'on doive faire ça sur cette prop). Là ça passe chez moi en tout cas ^^.